riposte-core/src/main/java/com/nike/riposte/util/AsyncNettyHelper.java

```diff
@@ -505,4 +509,80 @@ public static boolean executeOnlyIfChannelIsActive(ChannelHandlerContext ctx, St
             }
         }
     }
+
+    /**
+     * Helper method for creating a `CompletableFuture` that is using the tracing helpers.
+     * <p>
+     * <pre>
+     * AsyncNettyHelper.executeAsyncCall(() -> {
+     *      //do some work in a background thread
+     *      return VOID;
+     * }, executor, ctx);
+     * </pre>
+     */
+    public static <U> CompletableFuture<U> executeAsyncCall(Supplier<U> f, Executor executor, ChannelHandlerContext ctx) {
```

Comment:
I'd like to reserve `execute...` method names for helper methods related to `Executor.execute(...)`. Since these all return `CompletableFuture`, how about `supplyAsync(...)` to be in line with `CompletableFuture.supplyAsync(...)`?

Reply:
You are correct in where I got the initial names. After thinking on it more, i see why you prefer supplyAsync()... I'll adjust the naming to match the same. Great suggestion.